NEWS.md

```diff
@@ -1,3 +1,8 @@
+# hubUtils 0.1.1
+
+* Remove `hubAdmin` Suggests dependency by moving test hub configuration validation to CI.
```

Comment:
I always like to give credit and link to the PR in the news so that it's clear where different features came from. 

```suggestion
* Remove `hubAdmin` Suggests dependency by moving test hub configuration validation to CI
  (resolved: @annakrystalli, https://github.com/hubverse-org/hubUtils/issues/158)
```